simulate.m

```diff
@@ -88,7 +85,7 @@ function simulate(Param, DataIn, utilLo, utilHi)
 		resultsStore(iStation).power = pIn;
 		resultsStore(iStation).schedule = Stations(iStation).ScheduleDL;
 		
-		% Check utilisation metrics and change status if needed
+		% Check utilisation metrics and change PowerState if needed
 		Stations(iStation) = checkUtilisation(Stations(iStation), utilPercent,...
```

Comment:
Could this be renamed? `checkUtilisation` is very passive. I'm guessing it sets a power state based on utilisation. So maybe `computePowerState`?

Reply:
Changed to evaluatePowerState